.github/workflows/continuous-integration.yaml

```diff
@@ -62,3 +62,23 @@ jobs:
 
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v1
+
+  api-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@v2.4.0
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Run api-docs
+        run: |
+          make api-docs
+      - name: Check if working tree is dirty
```

Comment:
On second thought, I think we need to call `make api-docs` from within the `generate` command and use the `ensure-generate-is-noop` for this check.